qlib/finco/task.py

```diff
@@ -648,3 +702,38 @@ def get_figure_path(self):
     def save_markdown(self, content: str):
         with open(Path(self.workspace).joinpath(self.__DEFAULT_REPORT_NAME), "w") as f:
             f.write(content)
+
+
+class BackForwardTask(Task):
```

Comment:
When we fine-tune the prompt, we don't have to define it as an instance of `Task`. 

The prompt fine-tuning is supposed to be a learning workflow at an upper level. So, they do not necessarily share the same concepts.